src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1487,9 +1492,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Implements IVariableDeclarationStatement
 
             Private ReadOnly _variables As ImmutableArray(Of IVariableDeclaration)
+            Private ReadOnly _syntax As SyntaxNode
```

Comment:
No, that `Syntax` is from `BoundNode`, http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/BoundTree/BoundNode.cs,4494ab96c2bc2ab4. Making this more specific won't allow me to remove that cast.